packages/automation/src/Dummy/Dummy.stories.js

```diff
@@ -7,8 +7,10 @@
 
 import React from 'react';
 
-export const Default = () => <>Story</>;
+import { sectionTitle } from '../../config';
+
+export const Default = () => <>Dummy Story</>;
 
 export default {
-  title: 'Automation/Story',
+  title: `${sectionTitle}/Dummy`,
```

Comment:
Is there still a way for us to unify the formatting for these stories centrally in `core`? For example, while the section title might live in each individual package, we can still ensure that the `${sectionTitle}/Story` string is consistent across each.

Reply:
i cannot think of an easy / straightforward way really without creating overhead of this relatively simple formatter